traffic_ops/testing/api/v4/regions_test.go

```diff
@@ -142,6 +161,38 @@ func SortTestRegions(t *testing.T) {
 	}
 }
 
+func SortTestRegionsDesc(t *testing.T) {
+
+	respAsc, _, err1 := TOSession.GetRegions(nil, nil)
+	params := url.Values{}
+	params.Set("sortOrder", "desc")
+	respDesc, _, err2 := TOSession.GetRegions(params, nil)
+
+	if err1 != nil {
+		t.Errorf("Expected no error, but got error in Regions Ascending %v", err1)
+	}
+	if err2 != nil {
+		t.Errorf("Expected no error, but got error in Regions Descending %v", err2)
+	}
+	if len(respAsc) == len(respAsc) {
+		if len(respAsc) > 0 && len(respDesc) > 0 {
+			// reverse the descending-sorted response and compare it to the ascending-sorted one
+			for start, end := 0, len(respDesc)-1; start < end; start, end = start+1, end-1 {
+				respDesc[start], respDesc[end] = respDesc[end], respDesc[start]
+			}
+			if respDesc[0].Name != "" && respAsc[0].Name != "" {
+				if !reflect.DeepEqual(respDesc[0].Name, respAsc[0].Name) {
+					t.Errorf("Regions responses are not equal after reversal: %s - %s", respDesc[0].Name, respAsc[0].Name)
+				}
+			}
+		} else {
+			t.Errorf("No Response returned from GET Regions using SortOrder")
+		}
+	} else {
+		t.Fatalf("Region response length are not equal %v %v", respAsc, respAsc)
```

Comment:
I think you wanted to print `respAsc` and `descAsc` here, not `respAsc` twice - but also maybe you only wanted to print the lengths?